docs/architecture/decoupled-mode.md

```diff
@@ -0,0 +1,87 @@
+# Decoupled Mode
+
+In **Decoupled mode** the primary domain points to the nodejs server, or serverless function, hosting Frontity. This is the site that visitors access directly in order to view the content.
+
+![](../.gitbook/assets/frontity-architecture%20%282%29%20%288%29%20%288%29.png)
```

Comment:
The latest version of every documentation repo doesn’t have a `.gitbook/assets` folder anymore. All images in docs should be linked from frontity.org